app/views/member_mailer/welcome_coach.html.haml

```diff
@@ -6,7 +6,11 @@
 
 %p
   - if @member.subscriptions.any?
```

Comment:
Is there any way to automatically detect whether someone is subscribed to the London chapter? Is this exposed at all in `@member.subscriptions`?

Reply:
there is a way but it requires hardcoding the string `london`. I think it opens up a can of worms we could not open up, e.g. then other chapters will want special instructions too, etc.

Reply:
We're already hardcoding the string `London`, in the paragraph of instructions we're adding for London users. I don't think this is any different?

Reply:
I think if it's not too difficult to change the text like @richardwestenra suggested depending on chapter subscribed to it would better as it seems a lot of wording, when in fact this is only relevant for 1 out of 17 chapters

Reply:
Being a little bit custom for london alone is ok I think :)

Reply:
Not really because what if, for whatever reason, the name of the London chapter changes? Then the code has to be updated here and it might be easy to forget and we'll be banging our head against the table trying to figure out why the wrong instructions are sending. Again, it's very unlikely to happen but it's not good practice. That said if someone else vouches for this solution I'm happy to implement it. @KimberleyCook what do you think?

Reply:
ok, can put this in but just keep in mind this might cause some issues in the future 😂

Reply:
We'll have to remember ha, at the level codebar is growing I don't think all text suits all chapters so a few exceptions for London I think is ok :)

Reply:
I agree, it's not ideal, but in this situation I think it's okay. Also if we just set it up with a conditional just for london (e.g. `@member.subscriptions === london` or whatever) then if the London chapter's name changes then worst case scenario, they'll just get the same text as before until we notice the error.

Reply:
I investigated this issue further and noticed a possible issue. The problem is when a user signs up they can subscribe to more than 1 chapter. So they can subscribe to both London and Brighton. What would the text be then in that case? Seems like to me to send tailored messages based on what chapter the user subscribed to would entail setting up some sort of conditional statement with a lot of edge cases. You'll also notice that a member can sign up and not subscribe to any chapters (and they would still receive this message). For now we are sending them this text: `We send out invitations to our events via email. Sign up from the subscriptions page" to receive invitations to our workshops.` Again, this is factually incorrect but on this case we don't even have the means to check for which chapter they have subscribed to as they have subscribed to none. Any thoughts of how we could get around this?

Reply:
Ahh yes the several chapters problem, good point I forgot about this.

Reply:
What about something like this (pseudocode)?

> You'll automatically receive email invitations to our workshops as soon as a workshop is open for RSVP.
**if (member.subscriptions.includes(london)) {**
    However, our London chapter doesn't send workshop invitations via email, so you won't receive email invites for regular London events. We open RSVPs at around 1pm on the Friday before the workshop, and you will be able to register through the workshop page on our website. To be notified when we open RSVPs, join the #london-chapters channel [on our Slack](http://codebar-slack.herokuapp.com/), or [follow us on Twitter](https://twitter.com/codebar).
**}**

Reply:
@richardwestenra yep, that looks good, let me look into implementing that

Reply:
@richardwestenra @KimberleyCook this is now ready to review